activities/management/commands/jslps_data_integration.py

```diff
@@ -23,49 +23,159 @@ def send_mail(self):
 
     def handle(self, *args, **options): 
         
-        print "Geography,jslps_geo"
-        call_command('jslps_geo')
-
-        print "Geography,jslps_geo"
-        call_command('mksp_jslps_geo')
-
-        print "Groups,jslps_groups"
-        call_command('jslps_groups')
-
-        print "Groups,jslps_groups"
-        call_command('mksp_jslps_groups')
+        try:
```

Comment:
no use of try except clause here as we are not throwing any exception

Reply:
This is discussed . Will not change